flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/debezium/DebeziumJsonFileSystemITCase.java

```diff
@@ -0,0 +1,148 @@
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+
+/**
+ * Test Filesystem connector with DebeziumJson.
+ */
+public class DebeziumJsonFileSystemITCase extends StreamingTestBase {
+
+	private static final List<String> EXPECTED = Arrays.asList(
+			"+I(101,scooter,Small 2-wheel scooter,3.14)",
+			"+I(102,car battery,12V car battery,8.1)",
+			"+I(103,12-pack drill bits,12-pack of drill bits with sizes ranging from #40 to #3,0.8)",
+			"+I(104,hammer,12oz carpenter's hammer,0.75)",
+			"+I(105,hammer,14oz carpenter's hammer,0.875)",
+			"+I(106,hammer,16oz carpenter's hammer,1.0)",
+			"+I(107,rocks,box of assorted rocks,5.3)",
+			"+I(108,jacket,water resistent black wind breaker,0.1)",
+			"+I(109,spare tire,24 inch spare tire,22.2)",
+			"-D(106,hammer,16oz carpenter's hammer,1.0)", // -U
+			"+I(106,hammer,18oz carpenter hammer,1.0)", // +U
+			"-D(107,rocks,box of assorted rocks,5.3)", // -U
+			"+I(107,rocks,box of assorted rocks,5.1)", // +U
+			"+I(110,jacket,water resistent white wind breaker,0.2)",
+			"+I(111,scooter,Big 2-wheel scooter ,5.18)",
+			"-D(110,jacket,water resistent white wind breaker,0.2)", // -U
+			"+I(110,jacket,new water resistent white wind breaker,0.5)", // +U
+			"-D(111,scooter,Big 2-wheel scooter ,5.18)", // -U
+			"+I(111,scooter,Big 2-wheel scooter ,5.17)", // +U
+			"-D(111,scooter,Big 2-wheel scooter ,5.17)"
+	);
+
+	private File source;
+	private File sink;
+
+	private void prepareTables(boolean isPartition) throws IOException {
+		byte[] bytes = readBytes("debezium-data-schema-exclude.txt");
+		source = TEMPORARY_FOLDER.newFolder();
+		File file;
+		if (isPartition) {
+			File partition = new File(source, "p=1");
+			partition.mkdirs();
+			file = new File(partition, "my_file");
+		} else {
+			file = new File(source, "my_file");
+		}
+		file.createNewFile();
+		Files.write(file.toPath(), bytes);
+
+		sink = TEMPORARY_FOLDER.newFolder();
+
+		env().setParallelism(1);
+	}
+
+	private void createTable(String name, String path, boolean isPartition) {
+		tEnv().executeSql(format("create table %s (", name) +
+				"id int, name string, description string, weight float" +
+				(isPartition ? ", p int) partitioned by (p) " : ")") +
+				" with (" +
+				"'connector'='filesystem'," +
+				"'format'='debezium-json'," +
+				format("'path'='%s'", path) +
+				")");
+	}
+
+	@Test
+	public void testNonPartition() throws Exception {
+		prepareTables(false);
+		createTable("source", source.toURI().toString(), false);
+		createTable("sink", sink.toURI().toString(), false);
+
+		tEnv().executeSql("insert into sink select * from source").await();
+		CloseableIterator<Row> iter = tEnv().executeSql("select * from sink").collect();
```

Comment:
Select some fields to trigger the projection push down? We can write one more column into the sink, e.g `UPPER(name)`, and then select out the `upper_name` instead of the `name.